index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<header id="header">
+  <link rel="stylesheet" href="styles.css">
+  <img id="header-img" src="">Logo</img></br>
+  <nav id="nav-bar">
+    
+      <a href="#features" class="nav-link">Features</a>&nbsp;&nbsp;&nbsp;
+      <a href="#video" class="nav-link"> How it works</a>&nbsp;&nbsp;&nbsp;
+      <a href="#pricing" class="nav-link">Pricing</a>
+    
+  </nav>
+  </header>
+  <h1>Handcrafted, home-made masterpieces</h1>
+
+ 
+<form id="form" action="https://www.freecodecamp.com/email-submit">
+  <input id = "email" placeholder="email" name="email" type="email" required></input></br></br>
+ <input id="submit" type="submit"></input>
+</form>
+ <video id="video" src="">Video</video>
+<div id="features">
+<div>Premium materials</div>
+<div>Fast Shipping</div>
+<div> Quality Assurance</div>
+</div></br></br>
+<div id="pricing">
+  <div>$600</div>
+  <div>$900</div>
+  <div>$1200</div>  
+</div>
```

Comment:
Kindly make "pricing" div an unordered list instead, and make the children list items.

Reply:
- [OPTIONAL] Kindly state the reason for your requests. And if possible point students to any resources that could be of help to the students.

Reply:
Hi @Qoosim, I added linters file, but there were no GitHub action checks. What do I do?

Reply:
I can see them now, never mind. Thank you so much @Qoosim 🙏🏾

Reply:
@Qoosim I might be wrong but I think this exercise is more about "code-review" than me rewriting the code I wrote during my trial project. Hence, for me to pass the linters check, I will have to start rewriting the trial project code. Do you advise I rewrite the trial code, cuz if I do, the code review I made will be futile?

Reply:
To support my stance, kindly see this bullet point from the project requirement 

"Copy a link to the pull request and ask for a code review in your Student Dashboard as usual. (The only difference will be that the Code Reviewer won't check your code but the review you performed - isn't it a code review inception? ;))"